### PR TITLE
PluginSettings: Make settings object/class public

### PR DESCRIPTION
**Pull request by palinek**
fixes lxde/lxqt#949

### CODE REVIEW

File: plugin-statusnotifier/statusnotifierwidget.cpp [hunk 1]
```diff
@@ -28,6 +28,7 @@
 
 #include "statusnotifierwidget.h"
 #include <QApplication>
+#include <QDebug>
```
Comment (paulolieuthier) [inline plugin-statusnotifier/statusnotifierwidget.cpp]: ?

File: plugin-quicklaunch/lxqtquicklaunch.cpp [hunk 2]
```diff
@@ -36,13 +36,13 @@
 #include <QFileInfo>
 #include <QLabel>
 #include <QMessageBox>
-#include <QSettings>
 #include <QToolButton>
 #include <QUrl>
-#include <QSettings>
+#include <QDebug>
```
Comment (paulolieuthier) [inline plugin-quicklaunch/lxqtquicklaunch.cpp]: Remove?

**Comment by paulolieuthier** [comment]
GTM.

File: plugin-statusnotifier/statusnotifierwidget.cpp (hunk 1 above)
Comment (palinek) [inline plugin-statusnotifier/statusnotifierwidget.cpp]: There are debugs in this file -> https://github.com/lxde/lxqt-panel/blob/public_settings/plugin-statusnotifier/statusnotifierwidget.cpp#L40, https://github.com/lxde/lxqt-panel/blob/public_settings/plugin-statusnotifier/statusnotifierwidget.cpp#L53

I didn't introduce the qDebug() messages in this PR.... just the cleaning raised need of this include

File: plugin-quicklaunch/lxqtquicklaunch.cpp (hunk 2 above)
Comment (palinek) [inline plugin-quicklaunch/lxqtquicklaunch.cpp]: ...the very same as the previous comment:
https://github.com/lxde/lxqt-panel/blob/public_settings/plugin-quicklaunch/lxqtquicklaunch.cpp#L72, https://github.com/lxde/lxqt-panel/blob/public_settings/plugin-quicklaunch/lxqtquicklaunch.cpp#L77, https://github.com/lxde/lxqt-panel/blob/public_settings/plugin-quicklaunch/lxqtquicklaunch.cpp#L94

File: plugin-statusnotifier/statusnotifierwidget.cpp (hunk 1 above)
Comment (paulolieuthier) [inline plugin-statusnotifier/statusnotifierwidget.cpp]: OK.

**Pull request merged**